### DISCUSSION TITLE
WalletConnectKotlinV2 InterruptedIOException in CoreClient.initialize

### DISCUSSION THREAD

**Discussion by ron-recyclefarm** [WalletConnect/walletconnect-monorepo#3648] [Developer Support]
Hi, 
I got InterruptedIOException while initialize kotlin sdk

CoreClient.initialize : [com.walletconnect.android](http://com.walletconnect.android/).internal.common.exception.GenericException: Error while connecting, please check your Internet connection or contact support: [java.io](http://java.io/).InterruptedIOException: timeout

It's highly unlikely the internet connection is the problem so I came here.
What would I do?

**Comment by glitch-txs** [comment]
hi, could you post this on [Kotlin repo](https://github.com/WalletConnect/WalletConnectKotlinV2/issues)? Thanks
